examples/http-external/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: azure-http-external
+runtime: nodejs
+description: An http Azure Function App implemented in PowerShell
```

Comment:
can we add an example of a multi-function app as well?

Reply:
Well, we don't have functions for that yet :)
We can add one function in the example itself for now, just to make sure API works.

Reply:
Added